doc/cephfs/createfs.rst

```diff
@@ -78,11 +78,14 @@ failed. So when a MDS daemon eventually picks up rank 0, the daemon reads the
 existing in-RADOS metadata and doesn't overwrite it. The flag also prevents the
 standby MDS daemons to join the file system.
 
+The ``set`` option allows to set multiple options supported by ``fs set``
+interface at the same time.
+
 For example:
 
 .. code:: bash
 
-    $ ceph fs new cephfs cephfs_metadata cephfs_data
+    $ ceph fs new cephfs cephfs_metadata cephfs_data set max_mds 2 allow_new_snaps true
```

Comment:
`allow_new_snaps` is always true now. Maybe `allow_standby_replay` would be better.

Reply:
done